packages/ui/env.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+REQUIRED_VARIABLES=(
+  EXCHANGE_WALLET_PUB
+  GOOGLE_MAPS_API_KEY
+  ISSUER_ID
+  SMART_METER_ID
+  REGISTRATION_MESSAGE_TO_SIGN
+)
```

Comment:
There are some references in the code to variables:

NX_GOOGLE_MAPS_API_KEY
NX_EXCHANGE_WALLET_PUB
NX_REGISTRATION_MESSAGE_TO_SIGN
NX_SMART_METER_ID

Should these also be included here?

Reply:
As I have mentioned in PR description our UI env variables are now split between 2 sources. 
1. `env.sh` handles the secrets part which is added on build time and also adds `NX_` prefixes.

2. `dev-env.sh` and `prod-start.sh` are responsible for adding 3 other variables:
BACKEND_URL
BLOCKCHAIN_EXPLORER_URL
SUPPORTED_NETWORK_IDS
They can be adjusted on the runtime and they are publicly available through `window.config` since there is no reason in hiding them, but we can use this and change them at runtime.